frontend/src/components/LinkInformations/RedeemInformation.vue

```diff
@@ -1,11 +1,12 @@
 <template>
   <div class="redeem-information">
     <b-jumbotron bg-variant="muted" text-variant="dark" border-variant="info">
-      <h1 v-if="isContributionLink">
+      <h1 v-if="amount === ''">{{ $t('gdd_per_link.redeemlink-error') }}</h1>
+      <h1 v-if="isContributionLink && amount !== ''">
         {{ CONFIG.COMMUNITY_NAME }}
         {{ $t('contribution-link.thanksYouWith') }} {{ amount | GDD }}
       </h1>
-      <h1 v-else>
+      <h1 v-if="!isContributionLink && amount !== ''">
```

Comment:
What about putting the conditions (amount size, kind of link) in a method instead?

Reply:
at this point only data is transmitted, checked and displayed. 
the data is in the props and will never change. 
additional code would complicate things unnecessarily in my view. 
methods must then also be tested again.

Reply:
Well, then there is no need to change this.